File: CS/WindowsApplication3/Views/CustomerView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Mvvm;
using DevExpress.XtraEditors;

namespace DXSample.MVVM {
    [DevExpress.Utils.MVVM.UI.ViewType("CustomerView")]
    public partial class CustomerView : DevExpress.XtraEditors.XtraUserControl {
        public CustomerView() {
            InitializeComponent();
            if (!DesignMode) {
                var fluentApi = mvvmContext1.OfType<CustomerViewModel>();
                fluentApi.SetBinding(gridControl1, g => g.DataSource, x => x.Entities);
            }
        }
    }
}

File: VB/WindowsApplication3/Views/CustomerView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.Mvvm
Imports DevExpress.XtraEditors

Namespace DXSample.MVVM
    <DevExpress.Utils.MVVM.UI.ViewType("CustomerView")> _
    Partial Public Class CustomerView
        Inherits DevExpress.XtraEditors.XtraUserControl

        Public Sub New()
            InitializeComponent()
            If Not DesignMode Then
                Dim fluentApi = mvvmContext1.OfType(Of CustomerViewModel)()
                fluentApi.SetBinding(gridControl1, Function(g) g.DataSource, Function(x) x.Entities)
            End If
        End Sub
    End Class
End Namespace
